files/image_config/config-setup/config-setup

```diff
@@ -300,6 +300,16 @@ check_all_config_db_present()
     return 0   
 }
 
+# DB schema is subject to change between two images
+# Perform DB schema migration after loading backup config from previous image
+do_db_migration()
+{
+    if [[ -x /usr/local/bin/db_migrator.py ]]; then
```

Comment:
@judyjoseph  can you please review this part of change from multi DB namespace point of view.

With this change db_migrator will get called always. Earlier you ignored migration as part of https://github.com/sonic-net/sonic-buildimage/pull/4477

Why would db_migrator be skipped for multi db?

Reply:
@vaibhavhd the db_migrator should run fine for multi-asic config _db's too. I see the VERSIONS table with DATABASE version in multi-asic config _db's, hence it should work as it works for single asic config_db.

Reply:
Thanks, Judy! @yxieca we should be good to go ahead with this change. Can you please re-review.